src/framework/preferences/js/StarterSchemas.js

```diff
@@ -21,11 +21,41 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
      * contrast, table of contents, inputs larger and emphasize links
      *******************************************************************************/
 
+    var check;
+    var widthCheck = function() {
```

Comment:
Avoid using private variables and functions. Use a global namespace starting as "fluid.prefs.xxxx" for this function. Worthwhile to use context awareness for this check too.